strapi/strapi
# Fix: RBAC Same role as creator list view data display as - (dash)

Fix #10010 v4

### What does it do?

Fix RBAC bug in list view that show dash data for entries created by `same role as creator`. From debug, problem seems to came from casl v5 `permittedFieldsOf` not working as how it should be.
The fix change to pass subject as string instead of entry object that marked type by casl

### Why is it needed?

To fix list view not working as intended when config RBAC with `can read` + `same role as creator`

### How to test it?

1. Create fresh project quickstart / sqlite
2. Create a collection type model with a field, for example **title: string**
3. Create a new role `test`
4. Assign all field permissions and apply conditions
  - **Can Create**: blank
  - **Can Read**: `Is creator` and `Has same role as creator`
  - **Can Update**: `Is creator`
  - **Can Delete**: `Is creator`
5. Create 2 users with `test` role
6. Login as one, create an entry
7. Login as another one, see content in list view. You should see content normally



### Related issue(s)/PR(s)

Issue #10010 but for v4
